Pending feedback from#58

crsanti opened the pull request:
Solves #58.

crsanti committed:
Fixes for #65

crsanti committed:
Changed method names in validationEngine

crsanti committed:
Merge branch 'master' into #58-fix-pending-changes

nasdan commented:
- Rename `OnChange` eventsFilter to `onChange`
- In `fieldValidationEventFilter.spec` use annotation `result[0].eventsFilter["OnChange"]` to avoid TypeScript errors

crsanti committed:
Changed constant OnChange to onChange.
Changed some tests where accessing properties would not give intelisense (with atom-typescript it causes error).
Fixed typo.

crsanti committed:
Removed old TODOS

crsanti committed:
Merge branch 'master' into #58-fix-pending-changes
